shotover-proxy/Cargo.toml

```diff
@@ -49,7 +49,7 @@ rustls-pemfile = "1.0.2"
 aws-throwaway.workspace = true
 windsock = { path = "../windsock" }
 regex = "1.7.0"
-cql-ws = { git = "https://github.com/conorbros/cql-ws" }
+cql-ws = { git = "https://github.com/shotover/cql-ws" }
```

Comment:
oops

Reply:
Yeah, I'll just merge the other one first

Reply:
would be good to fix it here, to avoid the unnecessary dependency.